pages/_app.tsx

```diff
@@ -1,6 +1,6 @@
 import '~/styles/globals.css';
 import type { AppProps } from 'next/app';
-import { UserProvider } from '@auth0/nextjs-auth0';
+import { UserProvider } from '@auth0/nextjs-auth0/client';
```

Comment:
This is required after upgrading next auth, but we want to keep the current repo version of next auth, so no need to do this.